openapi/typescript.xml

```diff
@@ -0,0 +1,60 @@
+<project 
+    xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kubernetes</groupId>
+    <artifactId>client-javascript</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>client-typescript</name>
+    <url>http://kubernetes.io</url>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>2.2.2</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${generator.spec.path}</inputSpec>
+                            <language>typescript-node</language>
+                            <gitUserId>kubernetes-client</gitUserId>
+                            <gitRepoId>typescript</gitRepoId>
+                            <configOptions>
+                                <projectName>kubernetes-ts-client</projectName>
+                                <projectDescription>Typescript client for Kubernetes.</projectDescription>
+                                <projectVersion>${generator.client.version}</projectVersion>
+                                <projectLicenseName>Apache V2</projectLicenseName>
+                                <sortParamsByRequiredFlag>true</sortParamsByRequiredFlag>
+                                <supportsES6>false</supportsES6>
+                                <npmName>kubernetes-client-typescript</npmName>
```

Comment:
specially because we used typescript here, I suggest we use typescript instead of `ts` in project name.